contracts/GovHub.template

```diff
@@ -254,10 +255,10 @@ contract GovHub is System, IParamSubscriber {
       } else {
         callData = abi.encodePacked(bytes4(keccak256(bytes(proposal.signatures[i]))), proposal.calldatas[i]);
       }
-
-      (bool success, bytes memory returnData) = proposal.targets[i].call{ value: proposal.values[i] }(callData);
+      address _target = proposal.targets[i];
+      (bool success, bytes memory returnData) = _target.call{ value: proposal.values[i] }(callData); //@dev:unsafe  //@openissue
```

Comment:
not strictly a bug - call() will revert if needed, still I would add a check that balance >= proposal.values[i]